streams/runners/storm/layout/src/main/java/com/hortonworks/streamline/streams/layout/storm/CustomProcessorBoltFluxComponent.java

```diff
@@ -39,14 +39,13 @@
     protected void generateComponent () {
         String boltId = "customProcessorBolt" + UUID_FOR_COMPONENTS;
         String boltClassName = "com.hortonworks.streamline.streams.runtime.storm.bolt.CustomProcessorBolt";
-        String[] configMethodNames = {"customProcessorImpl", "outputSchema", "inputSchema", "config", "inputSchemaMap"};
+        String[] configMethodNames = {"customProcessorImpl", "outputSchema", "config", "inputSchemaMap"};
         Object[] values = new Object[configMethodNames.length];
         values[0] = conf.get(TopologyLayoutConstants.JSON_KEY_CUSTOM_PROCESSOR_IMPL);
         try {
             values[1] = getOutputSchemaJson();
```

Comment:
getInputSchemaJson is removed but not getOutputSchemaJson but we want to make both optional ?

Reply:
Those were config methods to the bolt. InputSchema to the bolt before was never used. So it was an unused instance variable in the bolt. Moreover, it did not make sense to pass it since its provided during CP registration initially. Hence removed that variable and the corresponding method. OutputSchema here is the output schema from the bolt configured by user in SAM app using UI. We need it to declare fields from the CustomProcessorBolt.